### DISCUSSION TITLE
v0.9.12

### DISCUSSION THREAD

**Discussion by flemming-n-larsen** [robocode-dev/tank-royale#1] [Announcements]
## About

This is the first release of Robocode Tank Royale.
This release is an early alpha version meant for people to try out and give feedback on.

## Documentation

You can read more about Robocode Tank Royale here:
https://robocode.dev/tankroyale/docs/

## Running Robocode

You need Java 11 or newer to be preinstalled on your system. You can read the [installation guide](https://robocode.dev/tankroyale/docs/articles/installation.html) to get more details about installing Java and Robocode.

The Robocode application (GUI) can be run from the command-line:

On Windows:
```
java -jar robocode-tankroyale-gui-0.9.12.jar
```
On macOS and Linux:
```
sudo java -jar robocode-tankroyale-gui-0.9.12.jar
```
Note that you need to download sample bots and install those into directories on your system, and add these directories from the menu: Config -> Bot Root Directories

## Sample bots

These sample bots are currently available:

C#: [sample-bots-csharp-0.9.12.zip](/robocode-dev/tank-royale/releases/v0.9.12/download/sample-bots-java-0.9.12.zip)
Java:  [sample-bots-java-0.9.12.zip](/robocode-dev/tank-royale/releases/v0.9.12/download/sample-bots-java-0.9.12.zip)

Note that the C# bots need Microsoft .Net SDK 5.0 or newer to be preinstalled.

## Bot API

In order to develop bots for Robocode, you'll need one of the provided APIs.
There is a Java version available with this release as a .jar archive: [robocode-tankroyale-bot-api-0.9.12.jar](/robocode-dev/tank-royale/releases/v0.9.12/download/robocode-tankroyale-bot-api-0.9.12.jar)
Later on, the Bot API will be made available on a Maven repository as well via Sonatype.

And the .Net version of the Bot API is available from Nuget here: https://www.nuget.org/packages/Robocode.TankRoyale.BotApi/

<hr /><em>This discussion was created from the release <a href='https://github.com/robocode-dev/tank-royale/releases/tag/v0.9.12'>v0.9.12</a>.</em>

**Comment by John-Paul-R** [comment]
The "installation guide" link in this release points to localhost. Should probably be https://robocode.dev/tankroyale/docs/articles/installation.html

**Reply by flemming-n-larsen** [reply]
Thank you for the notification. 🙂
I have fixed the broken link. 👍
